benchmarks/polys.py

```diff
@@ -162,5 +164,44 @@ class TimePREM_QuadraticNonMonicGCD(_TimePREM):
 
 
 class TimePREM_SparseNonMonicQuadratic(_TimePREM):
+    GCDExampleCLS = _SparseNonMonicQuadratic
+    params = [(1, 3, 5, 8), ('expr', 'dense', 'sparse')]
+
+
+class _TimeSUBRESULTANTS(_TimeOP):
+    """Benchmarks for pseudo-quotient method"""
+
+    def expected(self, f, g, d, syms):
+        x = syms[0]
+        subresultant = resultant(f * LC(g, x) ** (degree(f, x) - degree(g, x) + 1), g, x)
```

Comment:
I think this is the wrong equation

Reply:
@oscarbenjamin  Is there any formula that exists to check the correct result?

Reply:
I don't know if there is a formula. Checking against `subresultants` is fine though.